CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 ### Enhancements
 
 - Add ValidationFailed state and signal in anticipation of validating task outputs [#2143](https://github.com/PrefectHQ/prefect/issues/2143)
+- Add Result.validate method [#2144](https://github.com/PrefectHQ/prefect/issues/2144)
```

Comment:
Could we expand this entry a couple words to say either what the validate method is or what it does? Could be as simple as `Add Result.validate method to check validator functions` or something like that

Reply:
Fixed in 0bcfab0

Reply:
Fixed in 0bcfab0